### DISCUSSION TITLE
Temperature sensors

### DISCUSSION THREAD

**Discussion by szili83** [milo526/homebridge-tuya-web#544] [General]
I have a lot of temperature sensort, they connext to the Smart home. It is possible, that I can see them?

**Comment by fratinize** [comment]
This seems to be in tuyas court
I have a temp/humidity sensor TH01 shows fine in tuya and smart life but is not listed in tuya api list
I emailed tuya asking them to add

**Comment by rdouma** [comment]
I was looking for a reason why I don't see my new Afintek temperature and humidity sensors (I do see them fine in the Smart Life app). Does Tuya still not support it via the API, or do you perhaps have a URL to the API docs? Thanks!
